src/org/thoughtcrime/securesms/jobs/PushTextSendJob.java

```diff
@@ -57,7 +58,7 @@ public void onPushSend(MasterSecret masterSecret) throws NoSuchMessageException,
       deliver(record);
 
       String messageBody = record.getDisplayBody().toString();
-      if (messageBody.length() >= 19 && messageBody.substring(0, 19).equals("{\"type\": \"reaction\"")) {
+      if (!Stereotype.fromBody(messageBody).equals(Stereotype.UNKNOWN)) {
```

Comment:
Damn, `Stereotype` is such a convenient class wonder who the all-star pair programmers who wrote it are

Reply:
hahahah

Reply:
Yeah sick job guys!